dot/state/block.go

```diff
@@ -87,11 +87,20 @@ func NewBlockState(db database.Database, bt *blocktree.BlockTree) (*BlockState,
 
 	bs.genesisHash = bt.GenesisHash()
 	var err error
+
+	// set the current highest block
 	bs.highestBlockHeader, err = bs.BestBlockHeader()
 	if err != nil {
 		return nil, err
 	}
 
+	// set the latest finalized head to the genesis header
+	// TODO: will need to load and set this upon node startup to the actual hash stored in the db
```

Comment:
do we have an issue open for this? maybe add issue number to TODO comment

Reply:
I'll open an issue

Reply:
#898